src/Faker/Factory.php

```diff
@@ -25,6 +25,10 @@ public static function create($locale = self::DEFAULT_LOCALE)
         return $generator;
     }
 
+    /**
+     * @param string $locale
```

Comment:
why don't you document `$provider`, too? (string). Same for `findProviderClassname`